### PR TITLE
Added post-merge hook using husky

### LINKED ISSUE #524: Adding Post-merge functionality to husky as discussed on slack to improve developer's experience.
**Describe the bug**
Currently, we dont have any post-merge functionality in husky... Sometimes, when some new dependencies are added to package.json and user git pulls the files to his local system, he gets errors and some of the required node modules will be missing....To prevent this , we can add post-merge functionality so that everytime a user uses 'git pull', if the remote repo had some changes into package.json, 'yarn' should automatically run

**To Reproduce**

1.update your remote repo and then use git pull
2.u will see this error if the remote had some changes in pakage.json

**Expected behavior**
no errors should be there and the developing experience should be good 

**Actual behavior**
might get errors sometime and developers may get stuck

**Screenshots**
not required

### PR DESCRIPTION
**Pull request by anurag0006**
**What kind of change does this PR introduce?**
a bugfix

**Issue Number:**
#524 
Fixes #

**Did you add tests for your changes?**
no

**Snapshots/Videos:**
 not required

**If relevant, did you update the documentation?**
Yes

**Summary**
The PR will improve developer experience and hence closes #524 

**Does this PR introduce a breaking change?**
No

### CODE REVIEW

**Comment by anurag0006** [comment]
@palisadoes, i had to create a new file for 'post-merge'...This is making some of the checks to fail...What should i do?

**Comment by palisadoes** [comment]
- Please fix the first comment so that the issue automatically closes. The PR_GUIDELINES.md file has details.
- Accessing the restricted files is OK for this PR

File: INSTALLATION.md [hunk 1]
```diff
@@ -174,6 +174,12 @@ yarn lint:fix
 yarn typecheck
 ```
 
+We are also using the package `Husky` to automatically run the following scripts on your changes whenever you make a pull from your remote repository:(This hook will only run if there are changes in pakage.json file)
```
Comment (xoldyckk) [inline INSTALLATION.md:177]: Make two seperate sub sections for pre-commit and post-push hooks explaining what they're doing in detail. Also provide example for both on how to opt out of executing those hooks.

**Review by xoldyckk — APPROVED** [review]

**Comment by xoldyckk** [comment]
@anurag0006 Just copy paste the same stuff from talawa-api and paste here.

**Comment by anurag0006** [comment]
@xoldyckk  i thought of doing that , but here it was written differently with some more extra information so i wrote it in another way....here also i have created two subsections as told and i think it is looking okk....but if you think using the lines writen in api would be better, i will do that :
So should i copy paste that one?

**Comment by xoldyckk** [comment]
> @xoldyckk i thought of doing that , but here it was written differently with some more extra information so i wrote it in another way....here also i have created two subsections as told and i think it is looking okk....but if you think using the lines writen in api would be better, i will do that : So should i copy paste that one?

Yes, do so. I left a few more comments on api fix those.

**Comment by palisadoes** [comment]
@anurag0006 Let me know when you are done so we can merge.

**Comment by anurag0006** [comment]
@palisadoes the required changes have been made...Please merge this PR

**Review by xoldyckk — APPROVED** [review]

**Review by palisadoes — APPROVED** [review]

**Pull request merged**